src/scripts/download_ftspeech.py

```diff
@@ -0,0 +1,80 @@
+"""Script that downloads the FTSpeech corpus.
+
+Usage:
+    python download_ftspeech.py <download_dir>
+"""
+
+try:
+    from selenium import webdriver
+    from selenium.webdriver.common.by import By
+    from selenium.webdriver.firefox.options import Options
+except ImportError:
+    raise ImportError(
+        "To run this script, `selenium` needs to be installed. Please install it with "
+        "`pip install selenium` or `poetry add selenium`."
+    )
+
+import time
+from getpass import getpass
+from pathlib import Path
+
+import click
+
+
+@click.command("Downloads the FTSpeech corpus.")
+@click.argument("download_dir", type=click.Path())
+def main(download_dir) -> None:
+    """Downloads the FTSpeech corpus."""
+    download_dir = Path(download_dir)
+
+    options = Options()
+    options.add_argument("--headless")
+    options.set_preference("browser.download.folderList", 2)
+    options.set_preference("browser.download.manager.showWhenStarting", False)
+    options.set_preference("browser.download.dir", str(download_dir))
+    options.set_preference(
+        "browser.helperApps.neverAsk.saveToDisk", "application/x-gzip"
+    )
+    driver = webdriver.Firefox(options=options)
+
+    # Go to the FTSpeech download page
+    driver.get("http://130.226.140.126")
+
+    # Get the access token input field
+    access_token_input = driver.find_element(By.XPATH, '//input[@name="token"]')
+
+    # Get the access token
+    access_token = getpass("Access token: ")
```

Comment:
So this prompts the user for the access-token, which is obtained from the FTSpeech download page form. I think it would be more clear to have either
1. the access-token as a flag. (Not so elegant as the token would be echoed in the terminal.)
2. Have a link to the form in the `getpass` prompt, something like:

```suggestion
    access_token = getpass("Enter access token (Which can be obtained here: https://docs.google.com/forms/d/e/1FAIpQLSdv0OweHtKh__u0J0M8cjtm9hg3Yo_y8bWxVjQLLUeEyCAxjw/viewform): ")
```

I know it looks like shit, but maybe its possible to embed links, idk? Anyway its a minor thing, so just skip it if its not easily fixable.

Reply:
Added it as a logging message before the access token is requested.